README.md

```diff
@@ -200,7 +210,7 @@ sudo /usr/local/bin/sandman --shutdown
 If you would like to run Sandman at boot, an init script is provided. You can start using it with the following commands:
 
 ```bash
-sudo cp sandman.sh /etc/init.d/
+sudo cp ~/sandman/sandman.sh /etc/init.d/
```

Comment:
Good catch. This does assume that the repository has been downloaded into the home directory, but this is more correct than what I had originally.